Userland/Libraries/LibWeb/CSS/StyleComputer.h

```diff
@@ -126,17 +133,29 @@ class StyleComputer {
 
     JS::NonnullGCPtr<DOM::Document> m_document;
 
+    struct AnimationKeyFrameSet {
+        struct ResolvedKeyFrame {
+            struct UseInitial { };
+            Array<Variant<Empty, UseInitial, NonnullRefPtr<StyleValue const>>, to_underlying(last_property_id) + 1> resolved_properties;
```

Comment:
This is uninitialized. Fixes https://github.com/SerenityOS/serenity/pull/19050#issuecomment-1564384089
```suggestion
            Array<Variant<Empty, UseInitial, NonnullRefPtr<StyleValue const>>, to_underlying(last_property_id) + 1> resolved_properties {};
```